kpi/maintenance_tasks.py

```diff
@@ -26,3 +26,16 @@ def remove_old_asset_snapshots():
         ).delete()
         if not count:
             break
+
+
+def remove_old_import_tasks():
+    delete_queryset = ImportTask.objects.filter(
+        date_created__lt=timezone.now() - timedelta(days=90),
+    )
+
+    while True:
+        count, _ = delete_queryset.filter(
```

Comment:
Now that we're using this a few times, we could consider adding abstraction. I'm fine doing this right now or in the next batch of work. 

`chunked_delete(queryset: QuerySet)`

Reply:
@bufke This will actually be the last PR for the KPI-portion of the Celery maintenance work. Should I just abstract this now or should we hold off until some theoretical future maintenance task tips it over into definite DRY territory?